pulp_smart_proxy/app/settings.py

```diff
@@ -2,3 +2,4 @@
 SMART_PROXY_AUTH_USERNAME = None
 SMART_PROXY_AUTH_PASSWORD = None
 SMART_PROXY_AUTH_METHODS = ["client_certificate"]
+SMART_PROXY_PULP_URL = None
```

Comment:
I saw there were checks. Would it make sense to enforce this is a valid URL, including protocol?

Reply:
left as an exercise to the reader, I have no idea how to do type checks inbetween django conf and dynafonf